### DISCUSSION TITLE
How to fix error when running docs app demo chat

### DISCUSSION THREAD

**Discussion by luandro** [langfuse/langfuse#1373] [Support] [answered]
### Describe the bug

When trying to use the chat get error from `supabaseClient.rpc`.

### To reproduce

Add all envs for qa chatbot, and run dev.

### Additional information

```
Failed to match page sections {
  code: 'PGRST106',
  details: null,
  hint: null,
  message: 'The schema must be one of the following: public, storage, graphql_public'
}
 ⨯ pages/api/qa-chatbot.ts (113:10) @ Object.handler
 ⨯ Failed to match page sections
  111 |     });
  112 |     console.error("Failed to match page sections", matchError);
> 113 |     throw new Error("Failed to match page sections");
      |          ^
  114 |   }
  115 |
  116 |   vectorStoreSpan.end({
```

**Comment by marcklingen** [comment]
Did you deploy the schema?

-- moving this to q&a to keep the issues focused on langfuse itself

**Reply by luandro** [reply]
Would be great to have a small section on the README on setting up Supabase. Unfortunately, I'm not so familiar as to know how to deploy a schema. Tried with the SQL command:
```
 CREATE SCHEMA docs;                                                                                                                                                                                                                          
                                                                                                                                                                                                                                              
 CREATE TABLE docs.documents (                                                                                                                                                                                                                
     id SERIAL PRIMARY KEY,                                                                                                                                                                                                                   
     title VARCHAR(255) NOT NULL,                                                                                                                                                                                                             
     content TEXT NOT NULL,                                                                                                                                                                                                                   
     created_at TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP,                                                                                                                                                                           
     updated_at TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP                                                                                                                                                                            
 );                                                                                                                                                                                                                                           
```

But that didn't do the trick. Would you mind pointing me in the right direction?

**Reply by marcklingen** [reply, marked as answer]
the docs q&a is just a little interactive demo of what langfuse can do. find details and all links on it here: https://langfuse.com/blog/qa-chatbot-for-langfuse-docs

I think you are looking for this: https://github.com/supabase/headless-vector-search

If you want to try langfuse (the product) you might want to use one of our cookbooks instead of deploying the q&a chatbot: https://github.com/langfuse/langfuse-docs/tree/main/cookbook

**Reply by luandro** [reply]
Awesome thanks! I thought those were all Python, so I was looking for a TS example. But now I see there's some TS examples there.

Would be amazing if there was a LlamaIndexTS example, that's what I've been looking for.

**Reply by marcklingen** [reply]
Sorry to disappoint you if you were looking for an integration with LlamaindexTS:, see note here https://langfuse.com/docs/integrations/llama-index/get-started

Feel free to add you upvote here: https://github.com/orgs/langfuse/discussions/1291

You can still use the Typescript SDK though if you know which kinds of events you would like to log in Langfuse.

**Reply by marcklingen** [reply]
Langfuse supports Langchain JS if that helps

**Reply by luandro** [reply]
Too much bootstrapping to get a Langchain RAG to the point of quality of LlamaIndex. But tomorrow I'll try to use same strategy as Python LlamaIndex in TS, should be doable right?

**Reply by marcklingen** [reply]
You can use our TS SDK, docs here: https://langfuse.com/docs/sdk/typescript/guide

example from which you might be able to copy paste some things: https://langfuse.com/docs/sdk/typescript/example-vercel-ai
